File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.lua
pos = 1
n = io.read("*number")
menor = io.read("*number")
for i=2,n do
    v = io.read("*number")
    if(v<menor) then
        menor = v
        pos = i
    end
end

print("Menor valor: "..menor);
print("Posicao: "..pos-1);

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1180.rb
n = gets.to_i
nums = gets.chomp().split(' ')
menor = (nums[0]).to_i
pos=1

for i in 1..(n-1) do
    if((nums[i]).to_i<menor) then
        menor = (nums[i]).to_i
        pos=i
    end
end
puts("Menor valor: #{menor}")
puts("Posicao: #{pos}")
